ui/partials/perf/comparerror.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert alert-danger" role="alert">
+    <p ng-repeat="error in errors">
+        <span class="fa fa-warning"></span> <span class="revision-holder">{{error}}</span>
```

Comment:
Yes, you'd need to separate out the revision from the rest of the error message to do this inside the compare controller. 

I think this is probably more trouble than it's worth at this point: the error dialog is pretty comprehensible as-is and hopefully it won't come up that often. :) Feel free to file a followup and work on it, but I'd rather move on to other things.